### ISSUE TITLE
Try to handle resources with bad ContentType (text/plain)

### ISSUE THREAD

**Issue by slorber** [read-write-web/rww-play#83]
For some existing RDF resources, the answer is returned with ContentType = text/plain

We could handle these resources by trying to parse them with different parsers maybe.

One exemple is:
http://kashori.com/JohnBlack/foaf.rdf

**Comment by slorber** [comment]
This now works fine thanks to this commit:

https://github.com/stample/rww-play/commit/31850b53f0fcb40fbe5158ed57a02724de733e4f

The following URL will work fine on next redeploy on stample.io:
https://www.stample.io/srv/cors?url=http://kashori.com/JohnBlack/foaf.rdf

while it currently gives error: `no Iteratee/parser for Content-Type Some(text/plain; charset=UTF-8) fetching http://kashori.com/JohnBlack/foaf.rdf`

**Comment by bblfish** [comment]
yes, but I am not too happy to be doing this. There is such a thing as content sniffing, to see if the first kb look like one type or the other. But it is part of the game of publishing on the web that people must follow the protocols.  Here the client is assuming it knows better than the publisher, which could become problematic. It becomes impossible to publish  text that looks like rdf. There are times where you exactly DON'T want it to be parsed as RDF ( for example if you want to make a statement about some RDF you found but disagree with)

So I am not sure what to do here. We should keep this open, so that a better solution can be found later. Better solutions would involve notifying the publisher of some RDF that his RDF is published with a broken mime type - or asking the user what he wishes to do.

If one is to do content-sniffing, we need to find out how to pass that information back to the client and how to deal with the above issues.
